### PR TITLE
Add setTransactionName() and ignoreTransaction() methods to BlackfireProbe stub

### PR DESCRIPTION
**Pull request by lolautruche**
These can be used when using Blackfire Monitoring.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

File: blackfire/blackfire.php [hunk 1]
```diff
@@ -104,4 +104,22 @@ public function close() {}
...
     public function createSubProfileQuery() {}
+    
+    /**
+     * Sets a custom transaction name for Blackfire Monitoring.
+     *
+     * @param string $transactionName Name to register for the transaction (e.g. 'user_model:show')
+     *
+     * @return void
+     */
+    public function setTransactionName(string $transationName) {}
```
Comment (isfedorov) [inline blackfire/blackfire.php:115]: Type in parameter name. That's why phpdoc is highlighted with "PhpDoc mismatch" inspection. Please correct the name.

**Review by isfedorov — CHANGES_REQUESTED** [review]
Please also invoke `composer cs-fix` to fix code style

File: blackfire/blackfire.php (hunk 1 above)
Comment (lolautruche) [reply blackfire/blackfire.php]: Thanks @isfedorov . Fixed now.

**Comment by isfedorov** [comment]
@lolautruche Thank you for the update but unfortunately cs-fixer still is failing. Command to fix code style is `docker-compose -f docker-compose.yml run php composer cs-fix -d /opt/project/phpstorm-stubs`. Also please ensure that your IDE doesn't reformat code before commit as it can reformat code the wrong way.

**Comment by lolautruche** [comment]
@isfedorov Sorry about that. I fixed the issue (hopefully).

**Pull request merged**